clients/docs/includes/configs/cloud/restproxy-sr.config

```diff
@@ -0,0 +1,16 @@
+# Kafka
+bootstrap.servers={{ BROKER_ENDPOINT }}
+security.protocol=SASL_SSL
+sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username="{{ CLUSTER_API_KEY }}" password="{{ CLUSTER_API_SECRET }}";
+ssl.endpoint.identification.algorithm=https
+sasl.mechanism=PLAIN
+client.bootstrap.servers={{ BROKER_ENDPOINT }}
+client.security.protocol=SASL_SSL
+client.sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username="{{ CLUSTER_API_KEY }}" password="{{ CLUSTER_API_SECRET }}";
+client.ssl.endpoint.identification.algorithm=https
+client.sasl.mechanism=PLAIN
+
+# Schema Registry specific settings
```

Comment:
Please follow comment consistency with the java.config file

Reply:
Similarly I copy-pasted it from the frontend code, but...

@LeslieKurt is consistency of the configuration file we give a problem? Namely, I'm wondering if it is okay for me to remove some configs or change some comments? The client connectivity will still work but the content will be slightly different from the ones we provide before.

Reply:
@cryoshida the whole point of the configs in this repo is that they represent the source of truth for what we recommend to users.  Therefore IMO it's important that they are consistent with what's already in _this_ repo for the other clients (e.g. Java client) and not for what is in the Cloud Console today.

Reply:
Sure thing! Since you stressed about the consistency, I updated other configuration's comments as well.